src/components/Playground-popup/index.js

```diff
@@ -0,0 +1,35 @@
+import React, { useState, useEffect } from "react";
+import PopupWrapper from "./popup.style";
+import popupImage from "./power-of-meshery.svg";
+import closeIcon from "./closeIcon.svg";
+import Button from "../../reusecore/Button";
+
```

Comment:
Let's wrap this in clickaway - https://www.npmjs.com/package/react-click-away-listener.